brackets-prisma-db/README.md

```diff
@@ -0,0 +1,39 @@
+# brackets-prisma-db
+
+This implementation of the [`CrudInterface`](https://drarig29.github.io/brackets-docs/reference/manager/interfaces/CrudInterface.html)
+uses [prisma](https://www.prisma.io/) to store the data in an SQL Database.
+
+> **Warning**
+> This implementation has not been full tested yet. Use with caution.
+
+## Limitations
+
+Currently there are some features of the [manager](https://github.com/Drarig29/brackets-manager.js) that can't be used.
+
+| **Feature**         | **Status**      |
+| ------------------- | --------------- |
+| Custom Participants | Implemented     |
+| Custom Matches      | Not Implemented |
```

Comment:
What do you mean by Custom matches? AFAIK, this is not something the manager supports 🤔

Reply:
Oh do you mean this? https://github.com/Drarig29/brackets-manager.js/commit/08fca03826a8a521ed7bfb7f2ec8327181d19c45

Reply:
Looks like it yeah :)

Just have seen this in the tests: https://github.com/Drarig29/brackets-manager.js/blob/master/test/custom.spec.js#L123
which looked to me like its possible to store extra field for matches.